.appveyor.yml

```diff
@@ -9,8 +9,8 @@ cache:
 
 environment:
     global:
-        CONDA_CHANNELS: conda-forge
```

Comment:
I'd rather move the parmed package to conda-forge

Reply:
I agree.

Can both be used together, with conda-forge higher priority? IIRC, conda's package resolving process gets slower and slower with more channels so we really try to minimize time there.

Reply:
Btw, omnia requires conda-forge https://github.com/ParmEd/ParmEd/blob/7e411fd03c7db6977e450c2461e065004adab471/devtools/ci/travis/install.sh#L25-L26

Reply:
The full dependency list from https://github.com/ParmEd/ParmEd/blob/7e411fd03c7db6977e450c2461e065004adab471/devtools/ci/travis/install.sh#L25-L39
```bash
    if [ -z "$MINIMAL_PACKAGES" ]; then
        # Install all prerequisites
        conda create -y -n myenv python=$PYTHON_VERSION \
            numpy scipy pandas nose openmm coverage nose-timer \
            netCDF4
        conda update -y -n myenv --all
        conda install -y -n myenv rdkit==2018.09.1
        conda install -y -n myenv boost==1.69.0
        conda install -y -n myenv nglview
        conda install -y -n myenv ambertools=18 -c ambermd
        conda install -y -n myenv networkx
        conda install -y -n myenv lxml
```
Note that it contains ambertools from the ambermd channel, rdkit, and boost as some heavy dependencies.

The minimal list https://github.com/ParmEd/ParmEd/blob/7e411fd03c7db6977e450c2461e065004adab471/devtools/ci/travis/install.sh#L41-L43 is essentially numpy:
```bash
        conda create -y -n myenv python=$PYTHON_VERSION numpy nose coverage \
            nose-timer
```

I don't really want any MDA user have to install these massive dependencies. But I also don't think one should create a crippled ParmEd conda-forge package (which does not pull all its dependencies) as this is a bad user experience for anyone who installs ParmEd for its own sake.

Should we make the ParmEdReader/Writer fail gracefully if no parmed package is installed? This would, however, go against our [importing rules](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#importing-modules ) for [modules in the core](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#list-of-core-module-dependencies).

Reply:
Regarding "guidelines" vs "rules": https://www.youtube.com/watch?v=WJVBvvS57j0 – should our rules be more something that we would call "guidelines"?

Reply:
Yes and (painfully) yes

`omnia` is a pretty well-maintained channel, although fairly specialized to the projects of the Pande, Chodera, Mobley, etc. labs. I'm not sure who is responsible for it but it also happens to be packaged on [`conda-forge`](https://anaconda.org/conda-forge/parmed). Compared to other dependencies, ParmEd is updated quite infrequently, so the risk of upstream complications should be fairly low (of course, there's also the downside of it possibly being slow to update to newer Python versions; not sure how that fits into your plans).

Reply:
@orbeckst parmed only requires `numpy`. And it's pretty easy to install via pip (or conda if exclude the requirements).

Reply:
That's good to know.

Thanks for comments, @mattihappy and @hainm !

Reply:
Looks like the conda-forge recipe only does linux for now, I raised an issue